manager-parent/manager-v2/src/main/java/com/redhat/service/smartevents/manager/v2/services/ProcessorServiceImpl.java

```diff
@@ -102,12 +103,6 @@ public Processor createProcessor(String bridgeId, String customerId, String owne
         return doCreateProcessor(bridge, customerId, owner, processorRequest);
     }
 
-    @Transactional
```

Comment:
I just moved this method as it was living in between two others that are related to each other.